guides/RELEASES.md

```diff
@@ -24,7 +24,7 @@ Release process
 
 ## 'Ready for release' column
 
-* Release branch (e.g. `release/1.7.3` or `beta/1.0.0`) taken from `develop`
+* Release branch (e.g. `release/1.7.0` see [version control][VERSIONING.md]) taken from `develop`
```

Comment:
`release/v1.7.0` maybe?

Reply:
I know we discussed this the other day but after looking at some repositories which have been updated (e.g. `dp-search-query`), I think maybe it should be `release/1.7.0`

Reply:
👍 happy with this approach